indra/newview/scripts/lua/LLChatListener.lua

```diff
@@ -0,0 +1,41 @@
+local fiber = require 'fiber'
+local inspect = require 'inspect'
+
+local LLChatListener = {}
+local waitfor = {}
+
+function LLChatListener:new()
+    local obj = setmetatable({}, self)
+    self.__index = self
+    obj.name = 'Chat_listener'
+
+    return obj
+end
```

Comment:
Heh - I want this to use `util.classctor()`, but that's still isolated on [my lua-login branch](https://github.com/secondlife/viewer/pull/1725/files#diff-867be21d2425e7bb99ec3554b881808693cb4d296bcbd55e3b04d173039f7215). We'll get there.